docs/topics/about-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
```

Comment:
Does this need its own module for IntelliJ? Seems to me that you could create a single "about-plugin" and substitute the plugin name locally. You might have to create an "ide-name" attribute. It seems like I have read this in all the other IDE guides.